vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/ItemClickListenerIT.java

```diff
@@ -79,6 +79,32 @@ public void doubleClickCheckboxInCell_noEventsFired() {
         checkbox.doubleClick();
         Assert.assertEquals("", getClickMessage());
         Assert.assertEquals("", getDoubleClickMessage());
+        Assert.assertEquals("", getColumnClickMessage());
+    }
+
+    @Test
+    public void clickCell_columnNameAvailable() {
+        grid.getCell(0, 0).click();
+        Assert.assertEquals("Name", getColumnClickMessage());
+    }
+
+    @Test
+    public void clickComponentInCell_columnNameAvailable() {
+        grid.getCell(0, 0).click();
+        TestBenchElement checkbox = grid.getCell(0, 1).$("vaadin-checkbox")
+                .first();
+        checkbox.click();
+        Assert.assertEquals("Name", getColumnClickMessage());
+    }
```

Comment:
The `ItemClickEvent` is specifically implemented in such a way that it doesn't fire when clicking focusable components, such as checkbox.

I'm not sure what is the purpose of this test. It's name implies that clicking the checkbox should fire event, but actually it doesn't. The only event comes from clicking the cell on the first column. This is already covered in the previous test.

There is already a test case for `clickCheckboxInCell_noItemClickEventFired`, so I think this case can be just removed.